internals/secrethub/service_ls.go

```diff
@@ -67,3 +93,50 @@ func (cmd *ServiceLsCommand) Run() error {
 
 	return nil
 }
+
+type serviceTable interface {
+	header() []string
+	row(service *api.Service) []string
+}
+
+type baseServiceTable struct{}
+
+func (sw baseServiceTable) header() []string {
+	return []string{"ID", "DESCRIPTION"}
+}
+
+func (sw baseServiceTable) row(service *api.Service) []string {
+	return []string{service.ServiceID, service.Description}
+}
+
+type keyServiceTable struct {
+	baseServiceTable
+}
+
+func (sw keyServiceTable) header() []string {
+	return append(sw.baseServiceTable.header(), "TYPE")
+}
+
+func (sw keyServiceTable) row(service *api.Service) []string {
+	return append(sw.baseServiceTable.row(service), string(service.Credential.Type))
+}
+
+type awsServiceTable struct {
+	baseServiceTable
+}
+
+func (sw awsServiceTable) header() []string {
+	return append(sw.baseServiceTable.header(), "ROLE", "KMS KEY")
```

Comment:
`KMS KEY` => `KMS-KEY`?

Reason is that an empty table now displays as:
```
ID  DESCRIPTION  ROLE  KMS KEY
```
Which suggests there are 5 columns instead of 4.